test/helper.py

```diff
@@ -124,8 +124,8 @@ def has_program(cmd, args=['--version']):
     """
     full_cmd = [cmd] + args
     for i, elem in enumerate(full_cmd):
-        if isinstance(elem, six.text_type):
-            full_cmd[i] = elem.encode(_arg_encoding())
+        if isinstance(elem, bytes):
+            full_cmd[i] = elem.decode(_arg_encoding())
```

Comment:
This converts everything to Unicode strings. Is that necessary on Python 3? If so, should we leave an `if PY2:` here to do the opposite on Python 2?